USAGE_android.md

```diff
@@ -361,6 +361,50 @@ in the GFXReconstruct GitHub repository at `layer/vk_layer_settings.txt`. Most
 binary distributions of the GFXReconstruct software will also include a sample
 settings file.
 
+#### Layer Settings via VkLayerSettingEXT
+
+A new way to configure the GFXReconstruct Vulkan capture layer is via the Vulkan
```

Comment:
nit: Prefer "alternate" to "new" since nobody is going to come back and remove the new when it isn't new anymore.